src/2023/location/index.html

```diff
@@ -133,10 +133,14 @@ <h2>Accommodation</h2>
 
           <p>Note that the rooms are limited, we encourage you to secure your reservation at your earliest convenience.</p>
 
-          <h2>Food and drinks</h2>
+          <h2>Food, drinks and sights</h2>
 
-          <p>Stay tuned for restaurant recommendations.</p>
+          <p>The next bar is located directly in the Progr. The <a href="https://www.progr.ch/de/im-progr/turnhalle/" target="_blank">Turnhalle</a> is directly below the Conference room. From 8:00 p.m. there will be music by Elis Noa. <a href="https://www.turnhalle.ch/de/programm/3298/" target="_blank">Tickets</a></p>
 
+          <p>Fresh air and good mulled wine are available at the <a href="https://bernersternenmarkt.ch/" target="_blank">Berner Sternenmarkt</a>. The 80 wooden huts serve winter delicacies, hot drinks and art. From the European promenade right next to the Federal Palace there is a remarkable view over the Marzili to the Alps.Fresh air and good mulled wine are available at the <a href="https://bernersternenmarkt.ch/" target="_blank">Berner Sternenmarkt</a>. The 80 wooden huts serve winter delicacies, hot drinks and art. From the European promenade right next to the Federal Palace there is a remarkable view over the Marzili to the Alps.</p>
+
+          <p>Or how about a walk through Bern's old town after an intensive day at the conference. Through the arcades (Lauben) it always goes down a little to the bear pit. On the way at the <a href="https://www.openstreetmap.org/query?lat=46.94866&lon=7.45558" target="_blank">Point</a> you can see the city stream flowing upwards :-) The bears are probably in hibernation, but fresh beer is brewed in the <a href="https://altestramdepot.ch/en/Welcome" target="_blank">"Alten Tramdepot"</a>. The most beautiful view of the city of Bern is definitely in the <a href="https://www.bern.com/en/detail/the-rose-garden" target="_blank">Rose Garden</a>.</p>
```

Comment:
Could/should we try to translate "Altes Tramdepot" in parenthesis, i.e. "(Old tram depot)". 🤷

Reply:
The restaurant is called "Altes Tramdepot" even on the English website. I tried to clarified the location..